app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :users_id
+  belongs_to :post, class_name: 'Post'
+  after_save :update_comments_count
+
+  def update_comments_count
+    post.increment!(:comment__counter)
+  end
```

Comment:
- [OPTIONAL] Great job implementing the required changes from the previous reviewer. Kindly make this method to be `private`. It is considered good practice to declare callback methods as private. If left public, they can be called from outside of the model and violate the principle of object encapsulation.